epub-a11y-meta-guide/1.0/draft/index.html

```diff
@@ -1379,58 +1379,614 @@ <h5>Sign Language</h5>
+                    &lt;itemref idref="chap1" media-overlay="chap1-overlay" /&gt;
+                  &lt;/spine&gt;
+                &lt;/package&gt;</code>
+                        </pre>
+
+                        <p><strong>Example Content Document (<code>content.xhtml</code>) referenced by the package:</strong></p>
+                        <pre>
+                            <code>&lt;!DOCTYPE html&gt;
+                &lt;html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops"&gt;
+                &lt;head&gt;...&lt;/head&gt;
+                &lt;body&gt;
+                    &lt;p id="p1"&gt;To be, or not to be, that is the question:&lt;/p&gt;
+                    &lt;p id="p2"&gt;Whether 'tis nobler in the mind to suffer&lt;/p&gt;
+                    &lt;p id="p3"&gt;The slings and arrows of outrageous fortune,&lt;/p&gt;
+                &lt;/body&gt;
+                &lt;/html&gt;</code>
+                        </pre>
+
+                        <p><strong>Example Media Overlay (<code>overlay.smil</code>) referenced by the package:</strong></p>
+                        <pre>
+                            <code>&lt;smil xmlns="http://www.w3.org/ns/SMIL" version="3.0"&gt;
+                  &lt;body&gt;
+                    &lt;seq id="seq1"&gt;
+                      &lt;par id="par1"&gt;
+                        &lt;text src="content.xhtml#p1"/&gt;
+                        &lt;audio src="audio/narration.mp3" clipBegin="0s" clipEnd="3.5s"/&gt;
+                      &lt;/par&gt;
+                      &lt;par id="par2"&gt;
+                        &lt;text src="content.xhtml#p2"/&gt;
+                        &lt;audio src="audio/narration.mp3" clipBegin="3.5s" clipEnd="6.2s"/&gt;
+                      &lt;/par&gt;
+                      &lt;par id="par3"&gt;
+                        &lt;text src="content.xhtml#p3"/&gt;
+                        &lt;audio src="audio/narration.mp3" clipBegin="6.2s" clipEnd="9.0s"/&gt;
+                      &lt;/par&gt;
+                    &lt;/seq&gt;
+                  &lt;/body&gt;
+                &lt;/smil&gt;</code>
+                        </pre>
+
+                        <p>The metadata to include in the package document would be:</p>
+                        <pre>
+                            <code>&lt;meta property="schema:accessibilityFeature"&gt;synchronizedAudioText&lt;/meta&gt;</code>
+                        </pre>
+                    </aside>
+                </section>
+                    
+                 <section id="metadata-a11y-features-timingControl">
+                    <h5>Timing Control</h5>
+
+                    <p>Indicates that the user can control the timing of any time-based interaction to meet their needs. This allows the user to turn off, adjust, or extend time limits on activities like tests or forms.</p>
+                    <p>This feature is critical for users who may require more time to read, comprehend, or respond to content.</p>
```

Comment:
It'd probably be good here to directly link this to [SC 2.2.1 Timing Adjustable](https://www.w3.org/TR/WCAG22/#timing-adjustable).